pvlib/tracking.py

```diff
@@ -569,3 +582,163 @@ def singleaxis(apparent_zenith, apparent_azimuth,
         out = {k: np.where(zen_gt_90, np.nan, v) for k, v in out.items()}
 
     return out
+
+
+def _get_rotation_matrix(angle, axis=0):
+    """
+    Return a rotation matrix that when multiplied by a column vector returns
+    a new column vector that is rotated clockwise around the given axis by the
+    given angle.
+
+    Parameters
+    ----------
+    angle : float
+        Angle of rotation [radians]
+    axis : int, default 0
+        Axis of rotation, 0=x, 1=y, 2=z
+
+    Returns
+    -------
+    rotation matrix
+
+    References:
+       `Rotation Matrix
+       <https://en.wikipedia.org/wiki/Rotation_matrix#Basic_rotations>`_
+
+    """
+    r11 = r22 = np.cos(angle)
+    r21 = np.sin(angle)
+    r12 = -r21
+    rot = np.array([
+        [1, 0, 0],
+        [0, r11, r12],
+        [0, r21, r22]])
+    rot = np.roll(rot, (axis, axis), (1, 0))
+    return rot
+
+
+def calc_tracker_axis_tilt(system_azimuth, system_zenith, tracker_azimuth):
+    """
+    Calculate tracker axis tilt in the global reference frame when on a sloped
+    plane.
+
+    Parameters
+    ----------
+    system_azimuth : float
+        direction of normal to slope on horizontal [radians]
+    system_zenith : float
+        tilt of normal to slope relative to vertical [radians]
+    tracker_azimuth : float
+        direction of tracker axes on horizontal [radians]
```

Comment:
>The fact that the descriptions of system_azimuth and system_zenith both refer to "slope" but not "system" might imply that they should be named `slope_azimuth` and `slope_tilt`.

Kevin, I like "slope" and I'm willing to change it. Is everyone else on board with using "slope" versus "system"? But I am a tiny bit ambivalent, because the slope might not be exactly the same as the plane that the trackers are in. Often we find the slope is not a plane, and so the support piles are driven or cut (or welded) with different reliefs to create a monoplane for the tracker axes. So should that plane still be called the "slope" or should it be called the "system". Maybe I should ask around the office. 

>Bit of a shame that these are in radians since all the other functions in a pvlib tracker angle workflow use degrees.

I'm ambivalent again. I understand the convenience of using degrees, and I also realize the errors that occur when radians are demanded, but it just irks me to have to convert back and forth, seems like a waste of computing time. I especially don't like the `cosd` and `sind` functions because they seem inefficient to me. My preference if this is required would be to change everything to radians once in near the top of the function, appending `_rad` as needed, and then perform the operations using `np.cos` and `np.sin`, then convert back to degrees if necessary at the output.